app/core/config.py

```diff
@@ -0,0 +1,108 @@
+"""
+Environments and configuration file
+"""
+import logging
+import os
+import secrets
+from enum import Enum
+from functools import lru_cache
+from typing import Optional
+from pydantic import BaseSettings
+
+logger = logging.getLogger(__name__)
+
+
+class EnvironmentEnum(str, Enum):
+    """
+    Enumerate the application environments
+    """
+
+    PRODUCTION = "production"
+    LOCAL = "local"
+
+
+class GlobalConfig(BaseSettings):
+    """
+    Base configuration to be modified by specific environments
+    """
+
+    TITLE: str = "Open Brewery DB REST API"
+    DESCRIPTION: str = "A REST API for Open Brewery DB"
+    JWT_SECRET: str = secrets.token_urlsafe(32)
+    ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 8
+    ALGORITHM = "HS256"
+    ENVIRONMENT: EnvironmentEnum
+    DEBUG: bool = False
+    TESTING: bool = False
+    TIMEZONE: str = "CDT"
```

Comment:
Just a suggestion, fine to leave it as is for now, but eventually let's make this UTC and/or add a note in the `README` so people know they need to change it.